src/licensedcode/data/licenses/st-microelectronics.yml

```diff
@@ -0,0 +1,8 @@
+key: st-microelectronics
+short_name: STMicroelectronics CentralLabs License
+name: STMicroelectronics CentralLabs License
+category: Permissive Free
```

Comment:
Since it has some serious restrictions on redistribution of graphics, I wonder if the category should be something else. @DennisClark what do you think?

Reply:
I am making this free restricted instead.... FWIW there is no such "Permissive Free" category anyway